content/docs/L/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "L"
+linkTitle: "L"
+weight: 4
+description: >
+ Load Balancing
+---
+
+<!-- {{% pageinfo %}}
+Load Balancing
+{{% /pageinfo %}} -->
+
+### What is Load Balancing?
+
+Imagine you’re in charge of a complex website, and it’s an online hit! It continues to face high amounts of traffic and you’re not sure your website backend can handle the amount of traffic from all over the world. You know you need to use load balancers, but the options are confusing. It can be hard to know exactly how to settle on a load balancing architecture that meets your needs, and figure out the prerequisites you need, for the best performance, without making too much of a dent in your wallet.
+To start, you need to know what load balancing is and why it’s so important to the long-lasting success of your application.
+
+>Load balancing is the process of distributing traffic across your network of servers to ensure that the system does not get overwhelmed and all requests are handled easily and efficiently.
+> 
+
+Modern high‑traffic websites serve hundreds of thousands, if not millions, of concurrent requests from users or clients and return the correct text, images, video, or application data, all in a fast and reliable manner. You’ve probably all experienced visiting your favorite website, only to get long wait times, connection timeout errors, or images and videos buffering. And a lot of the times, this is because the website backend is unable to cost‑effectively scale to meet these high volumes.
+The logical answer here is to add more backend servers to help serve traffic. But the next question becomes, how do you distribute traffic to those backend servers based on capacity and health?
+This is where load balancing makes a splash.
+
+In the seven-layer Open System Interconnection (OSI) model, network firewalls are at levels one to three (L1-Physical Wiring, L2-Data Link and L3-Network). Meanwhile, load balancing happens between layers four to seven (L4-Transport, L5-Session, L6-Presentation and L7-Application).
```

Comment:
Can we have such a pictorial representation for this section? Some may not be acquainted with the OSI model

![image](https://user-images.githubusercontent.com/53428217/163528275-56072b7f-5177-4a1e-9010-56806c9e6236.png)